server/src/main/java/org/elasticsearch/common/settings/SecureSetting.java

```diff
@@ -97,6 +97,23 @@ public T get(Settings settings) {
         }
     }
 
+    /**
+     * Returns the digest of this secure setting's value. This method can be called at all times, there is no requirement that
+     * {@code SecureSettings} be open, unlike {@code #get(Settings)}. The digest is used to check for changes of the value, without
```

Comment:
> there is no requirement that {@code SecureSettings} be open

This doesn't seem correct. The entries within the keystore wrapper are not filled in until the keystore has been decrypted.

Reply:
You're right, I have changed the description. My intention was to convey that the *hash* is accessible even after the close, unlike the *value* .